### PR TITLE
Updated Dockerfile

### PR DESCRIPTION
**Pull request by tSudharshan**

### CODE REVIEW

**Comment by bojmil-armedia** [comment]
Solr Cloud uses Zookeper. Do we have a Zookeper instance that will be used?

File: Dockerfile [hunk 1]
```diff
@@ -1,36 +1,48 @@
...
+    yum install -y lsof
+WORKDIR /opt
+ADD ${SOLR_URL} /opt/
+
+RUN set -ex;\
+    tar -xzvf solr-${SOLR_VERSION}.tgz;\
+    mv solr-${SOLR_VERSION} solr;\
+    rm solr-${SOLR_VERSION}.tgz;\
+    rm -f /opt/solr/server/solr/configsets/_default/conf/managed-schema /opt/solr/*.txt ;\
+    #rm -Rf /opt/solr/example /opt/solr/docs ;\
```
Comment (bojmil-armedia) [inline Dockerfile]: remove commented line if not needed.

**Comment by naveen-armedia** [comment]
> will be used?

@bojmil-armedia  solr cloud will run an embedded zookeeper if no external zookeeper detail is not provided.

**Comment by bojmil-armedia** [comment]
Internal Zookeper to the Solr instance? If so, we should have an external instance to be able so scale Solr. We do have few instances with scaled Solr instances in production.

**Review by david-oc-miller — APPROVED** [review]

**Comment by david-oc-miller** [comment]
We do need a Zookeeper container, and when Solr pods are started or stopped, these changes should be registered with Zookeeper.  No matter how many Solr pods are started, we only need 3 Zookeeper pods to keep track of everything.

**Pull request merged**